proposals/index.js

```diff
@@ -12,22 +12,85 @@ import { Block } from "@near-lake/primitives";
  *
  * @param {block} Block - A Near Protocol Block
  */
+
 async function getBlock(block: Block) {
-  const devhubOps = getDevHubOps(block);
+  const rfpOps = getRFPOps(block);
+  const proposalOps = getProposalOps(block);
+
+  const rfpOpsLen = rfpOps.length;
+  const proposalOpsLen = proposalOps.length;
 
-  if (devhubOps.length > 0) {
-    console.log({ devhubOps });
+  if (rfpOpsLen > 0 || proposalOpsLen > 0) {
+    const authorToRFPId = buildAuthorToRFPIdMap(block);
     const authorToProposalId = buildAuthorToProposalIdMap(block);
     const blockHeight = block.blockHeight;
     const blockTimestamp = block.header().timestampNanosec;
-    await Promise.all(
-      devhubOps.map((op) =>
-        indexOp(op, authorToProposalId, blockHeight, blockTimestamp, context)
-      )
-    );
+    try {
+      await Promise.all(
+        rfpOps.map((op) =>
+          indexRFPsOp(op, authorToRFPId, blockHeight, blockTimestamp, context)
+        ).concat(
+          proposalOps.map((op) =>
+            indexProposalsOp(op, authorToProposalId, blockHeight, blockTimestamp, context)
+          ))
+      );
+    } catch (error) {
+      console.error('Error processing block operations:', error);
+    }
   }
 }
 
+function getAddOrEditObject(block, startsWith) {
+  const stateChanges = block.streamerMessage.shards
+    .flatMap((e) => e.stateChanges)
+    .filter(
+      (stateChange) =>
+        stateChange.change.accountId === "truedove38.near" &&
```

Comment:
Don't forget to update the account back to `devhub.near` (here and in some other places)